draft-ietf-tls-tls13.md

```diff
@@ -5475,16 +5475,16 @@ TLSPlaintext.legacy_record_version MUST be ignored by all implementations.
 The value of TLSCiphertext.legacy_record_version MAY be ignored,
 or MAY be validated to match the fixed constant value.
 Version negotiation is performed using only the handshake versions
-(ClientHello.legacy_version,
-ClientHello "supported_versions" extension, and ServerHello.version).
+(ClientHello.legacy_version, as well as
+ClientHello and ServerHello "supported_versions" extension).
 In order to maximize interoperability with older endpoints, implementations
 that negotiate the use of TLS 1.0-1.2 SHOULD set the record layer
 version number to the negotiated version for the ServerHello and all
 records thereafter.
 
 For maximum compatibility with previously non-standard behavior and misconfigured
 deployments, all implementations SHOULD support validation of certification paths
-based on the expectations in this document, even when handling prior TLS versions'
+based on the expectations in this document, even when handling prior TLS versions
```

Comment:
I don't think this change is correct -- "versions'" with a trailing apostrophe is the possessive form of the plural "versions", i.e., "the handshakes of prior versions of TLS".

Reply:
@kaduk you're correct, thank you.